src/sass/_bootstrap-variables.scss

```diff
@@ -1440,5 +1440,24 @@ $kbd-bg:                            $gray-900 !default;
 
 $pre-color:                         null !default;
 
+// CUSTOM VARIABLES
+//
+// Sidebar
+$sidebar-width-open:    235px !default;
```

Comment:
out of curiosity, why are these values absolute values and not calculated based on an more global variable? For instance `5 * $spacer`